client/me/billing-history/upcoming-charges-table.jsx

```diff
@@ -30,7 +30,7 @@ class UpcomingChargesTable extends Component {
 		return (
 			<div className="billing-history__transaction-links">
 				<a href={ managePurchase( siteSlug, transaction.id ) }>
-					{ translate( 'Manage Purchase' ) }
+					{ translate( 'Manage purchase' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **32** times:
`translate( 'Manage Purchases', { context: 'button label'} )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/card-headings-form-labels-sentence-case)